### PR TITLE
Enable gpups run on rec model

### PR DESCRIPTION
**Pull request by pangengzheng**
### PR types

Others
### PR changes

Others
### Describe

Enable gpups run on rec model

### CODE REVIEW

File: python/setup.py.in [hunk 1]
```diff
@@ -593,9 +593,11 @@ if '${WITH_CINN}' == 'ON':
 
 if '${WITH_PSLIB}' == 'ON':
     shutil.copy('${PSLIB_LIB}', libs_path)
+    shutil.copy('${JVM_LIB}', libs_path)
```
Comment (risemeup1) [inline python/setup.py.in:596]: setup.py也要对应同步改动，可参考https://github.com/PaddlePaddle/Paddle/pull/51082

**Review by zyfncg — DISMISSED** [review]
LGTM for setup.py.in

**Review by YuanRisheng — DISMISSED** [review]

File: python/setup.py.in (hunk 1 above)
Comment (pangengzheng) [reply python/setup.py.in:596]: ok

**Review by risemeup1 — APPROVED** [review]
LGTM

**Review by zyfncg — APPROVED** [review]
LGTM for setup.py.in

**Pull request merged**